### DISCUSSION TITLE
Multi Device Compatibilty

### DISCUSSION THREAD

**Discussion by shreyshrey71** [timschneeb/GalaxyBudsClient#228] [Q&A]
Can there be a feature to register more than one device with the manager. I have 2 buds plus devices which I use alternatively when 1 runs out of battery, but I have to unregister and register each time to view in the manager.

**Comment by timschneeb** [comment]
This is already on my todo-list since someone else proposed this here: https://github.com/ThePBone/GalaxyBudsClient/issues/214

I doubt I will start working on this anytime soon because it would require me to rewrite large chunks of the current code base, as I don't have too much time to work on this app at the moment.
